iis-wf/iis-wf-import/src/main/java/eu/dnetlib/iis/wf/importer/converter/ProjectToOrganizationConverter.java

```diff
@@ -39,30 +41,33 @@
 	 */
 	public ProjectToOrganizationConverter(ResultApprover resultApprover, byte[] relationColumnFamilyBytes) {
 		super(resultApprover);
+		Preconditions.checkNotNull(resultApprover);
+		Preconditions.checkNotNull(relationColumnFamilyBytes);
 		this.relationColumnFamilyBytes = OafHelper.copyArrayWhenNotNull(relationColumnFamilyBytes);
 	}
 
 	// ------------------------ LOGIC --------------------------
 
-	@Override
 	/**
 	 * Builds collection of {@link ProjectToOrganization} objects for given
 	 * hbase input.
 	 * 
 	 * @param hbaseResult full hbase record
 	 * @param resolvedOafObject resolved Oaf object
-	 * @return collection of {@link ProjectToOrganization} or null
-	 * @throws Exception
+	 * @return collection of {@link ProjectToOrganization}
+	 * @throws InvalidProtocolBufferException thrown when unable to deserialize object
 	 */
+	@Override
 	public Collection<ProjectToOrganization> buildObject(Result hbaseResult, Oaf resolvedOafObject)
```

Comment:
I wonder why you hide the fact that you actually return List? Can't see any added value in returning Collection instead of List, but can see problems. Think about it: you won't be able to get elements by index for example.